components/content-service/pkg/layer/provider.go

```diff
@@ -227,6 +227,103 @@ func (s *Provider) GetContentLayer(ctx context.Context, owner, workspaceID strin
 	return nil, nil, xerrors.Errorf("no backup or valid initializer present")
 }
 
+// GetContentLayerPVC provides the content layer for a workspace that uses PVC feature
+func (s *Provider) GetContentLayerPVC(ctx context.Context, owner, workspaceID string, initializer *csapi.WorkspaceInitializer) (l []Layer, manifest *csapi.WorkspaceContentManifest, err error) {
```

Comment:
NOTE: we could refactor these functions `GetContentLayer` and `GetContentLayerPVC` into one in the future.

Reply:
Indeed. Once PVC is stable and we switched over to it, old code will be removed. (so that we will have only one version of that function)